### DISCUSSION TITLE
v2.0.0 BETA 1

### DISCUSSION THREAD

**Discussion by Kovah** [Kovah/LinkAce#832] [Announcements]
<p align="center">
<img src="https://github.com/user-attachments/assets/80fe708e-8448-4e12-9c2f-f9751664b157"/>
</p>

### This is the first BETA release of LinkAce 2. 🥳 

It contains major new features. You can take a look at the features with some explanations in [**the changelog for version 2**](https://www.linkace.org/docs/v2/general/changelog/).

There's still stuff to do for me. But in the meantime, I encourage brave users to **try the new version**. Especially the upgrade from version 1 must be tested by _real people_. Please leave any feedback here.

If you want to test LinkAce 2, please consult the [dedicated upgrade guide](https://www.linkace.org/docs/v2/upgrade/from-v1/). 

<hr /><em>This discussion was created from the release <a href='https://github.com/Kovah/LinkAce/releases/tag/v2.0.0-beta1'>v2.0.0 BETA 1</a>.</em>

**Comment by tigerzioo** [comment]
Followed the instructions to upgrade LinkAce from 1.15.1 to V2. Everything looks good. All links, lists and tags are migrated without any errors. Looking forward to the V2 release. Thank you.

**Comment by vfrex** [comment]
Hello, I decided to kick the tires on on LinkAce after a look through the field of open source bookmark tools. Very impressed with LinkAce and how the project is run. Many thanks for the work that went into developing and shaping the tool.

My no-docker test setup for v1.15.1 with sqlite went smoothly. My no-docker test setup for 2.0.0 beta-1 with sqlite did not. Is no-docker on 2.0.0 with sqlite a "supported" configuration? If so, should I submit github issues under 2.0.0 beta-1 or is there somewhere else you want beta issues documented?

**Reply by Kovah** [reply]
Hey, yeah v2 should work even better with SQLite than v1. Please post any issues and mention that you are using the v2 beta in the report.
